draft-ietf-mls-protocol.md

```diff
@@ -1658,38 +1646,32 @@ the attacker enough information to mount an attack. If Alice asks Bob:
 by the ciphertext length to an adversary expecting Alice to provide Bob
 with a day of the week at some point in their discussion.
 
-Similarily to TLS 1.3, the MLS messages MUST be padded before AEAD encryption
-with zero-valued bytes. Upon AEAD decryption, the length field of the plaintext
-is used to compute the number of bytes to be removed from the plaintext to get the
-correct data.
-If the padding mechanism is used to improve protection against timing side-channels,
-it MUST be implemented in a "constant-time" at the MLS layer and above.
+Similarily to TLS 1.3, if padding is used, the MLS messages MUST be padded
+before AEAD encryption with zero-valued bytes. Upon AEAD decryption,
+the length field of the plaintext is used to compute the number of bytes
+to be removed from the plaintext to get the correct data.
+As the padding mechanism is used to improve protection against timing
```

Comment:
Is it used for that? IT's more about message size side chanels.

Reply:
Yeah, tired I guess : ) Fixed, Thanks !